winter/core/state_key.py

```diff
@@ -0,0 +1,9 @@
+import typing
+
+import dataclasses
+
+
+@dataclasses.dataclass(frozen=True)
+class StateKey:
```

Comment:
I suggest to rename `state`. Because state has a very abstract meaning and also implies that it will be being changed from time to time. Some suggestions: `metadata`, `properties`.

Reply:
Done